test/spec/controllers/eventRegistrations.spec.js

```diff
@@ -68,7 +68,7 @@ describe('Controller: eventRegistrations', function() {
     scope.editRegistrant(registrant);
     $httpBackend.flush();
 
-    expect(openModal).toHaveBeenCalled();
+    expect(openModal).toHaveBeenCalled(); /* eslint-disable-line */
```

Comment:
Why do we need to disable lint on these lines?

Reply:
So it wants us to use `.toHaveBeenCalledWith()` instead of `toHaveBeenCalled()` but the objects these expect are a giant mess and not worth the effort I think to add. As long as it's being called it should be okay.